.pullapprove.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
```

Comment:
Ah, it looks like develop got rebased onto your branch (instead of the other way around)